AdapterInterface.php

```diff
@@ -75,7 +75,7 @@ public function compareAutoRouteContent(AutoRouteInterface $autoRoute, $contentO
      *
      * @return null|Symfony\Cmf\Component\Routing\RouteObjectInterface
      */
-    public function findRouteForUri($uri);
+    public function findRouteForUri($uri, UriContext $uriContext);
```

Comment:
The `$uri` is the primary concern in this method, the `UriContext` may or may not be used to infer contextual things (f.e. if the route depends upon the subject document).